components/raftstore/src/store/fsm/peer.rs

```diff
@@ -6555,6 +6557,66 @@ where
     fn register_report_region_buckets_tick(&mut self) {
         self.schedule_tick(PeerTick::ReportBuckets)
     }
+
+    fn register_check_peer_complete_recovery(&mut self) {
+        self.schedule_tick(PeerTick::CheckPeerCompleteRecovery)
```

Comment:
How often does it tick?

Reply:
By default, it's a 10 seconds-level tick.

For simplification, it reuse the tick interval of `check_long_uncommitted_interval`, its setting is `10 seconds` by default.

And if one peer finds that it already finished `apply` stage (the log gap of `apply_index` reaches the limits), it also will stop ticking in advance.

Reply:
As the later implementation shows, the tick is deprecated and replaced with the normal check when `on_apply_res`. The later one is better.